terraform/common.tf

```diff
@@ -31,7 +31,7 @@ variable "sample_app_image" {
 }
 
 variable "aoc_image_repo" {
-  default = "public.ecr.aws/aws-otel-test/adot-collector-integration-test"
+  default = "public.ecr.aws/aws-observability/aws-otel-collector"
```

Comment:
Can we keep the image to - public.ecr.aws/aws-otel-test/adot-collector-integration-test integ test image itself as in by default to pick the latest test image. Not a strong reason though, just a preference

Reply:
done